edx_proctoring/templates/timed_exam/entrance.html

```diff
@@ -8,14 +8,14 @@
   <p>
     {% trans "This exam has a time limit associated with it." %}
     <strong>
-      {% trans "To pass this exam you must complete the problems in the time allowed." %}
+      {% trans "To pass this exam, you must complete the problems in the time allowed." %}
     </strong>
-    {% trans "As soon as you indicate that you are ready to start the exam, you will have "%} {{total_time|lower}} {% trans " to complete the exam." %}
+    {% trans "After you select to start the exam below, you will have "%} {{total_time|lower}} {% trans " to complete and submit the exam." %}
```

Comment:
This is just a little awkward - maybe this instead:

After you select **I am ready to start this timed exam**, you will have...